### PR TITLE
finish about page

### PR DESCRIPTION
**Pull request by havanhuy1997**
1. install all dependencies :  yarn
2. All css and js are bundled in folder assets/generated/ : yarn build:dev
The bundled css :  styles.bundle.css
The bundled js : scripts.bundle.js
3. Old base.html was renamed to base_sso.html
4. /about is url of about page

### CODE REVIEW

**Comment by havanhuy1997** [comment]
Please check

File: packages/webapp/node_modules/.cache/babel-loader/32fe88f93b73c6e05b5b6f33064caf85.json [hunk 1]
```diff
@@ -0,0 +1 @@
...Function(node: HtmlElement, isAppearing: bool) -> void\n   */\n  onEntered: PropTypes.func,\n\n  /**\n   * Callback fired before the \"exiting\" status is applied.\n   *\n   * @type Function(node: HtmlElement) -> void\n   */\n  onExit: PropTypes.func,\n\n  /**\n   * Callback fired after the \"exiting\" status is applied.\n   *\n   * @type Function(node: HtmlElement) -> void\n   */\n  onExiting: PropTypes.func,\n\n  /**\n   * Callback fired after the \"exited\" status is applied.\n   *\n   * @type Function(node: HtmlElement) -> void\n   */\n  onExited: PropTypes.func // Name the function so it is clearer in the documentation\n\n} : {};\n\nfunction noop() {}\n\nTransition.defaultProps = {\n  in: false,\n  mountOnEnter: false,\n  unmountOnExit: false,\n  appear: false,\n  enter: true,\n  exit: true,\n  onEnter: noop,\n  onEntering: noop,\n  onEntered: noop,\n  onExit: noop,\n  onExiting: noop,\n  onExited: noop\n};\nTransition.UNMOUNTED = 0;\nTransition.EXITED = 1;\nTransition.ENTERING = 2;\nTransition.ENTERED = 3;\nTransition.EXITING = 4;\n\nvar _default = (0, _reactLifecyclesCompat.polyfill)(Transition);\n\nexports.default = _default;","map":null,"metadata":{},"sourceType":"script"}
```
Comment (jbothma) [inline packages/webapp/node_modules/.cache/babel-loader/32fe88f93b73c6e05b5b6f33064caf85.json]: node_modules should not be checked into git.

You could add this to .gitignore if it helps
```
node_modules
```

File: _data/2018-19/manual_resources.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+budget_speech:
```
Comment (jbothma) [inline _data/2018-19/manual_resources.yaml]: Why is _data added to this repo? You shouldn't need it for the about page.

In fact, _data is generated from yaml views in this project. In future changes we will make the jekyll pages that depend on _data data use the django views for their context.

In this case, please delete _data.

File: .gitignore [hunk 3]
```diff
@@ -5,4 +5,6 @@ staticfiles/
 docs/_build
 *.sqlite3
 .tox
-.coverage
\ No newline at end of file
+.coverage
+/.vscode/
+/node_modules/
```
Comment (jbothma) [inline .gitignore]: I think if you remove the slashes it will also work for packages/webapp/node_modules

File: _layouts/about.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{% extends 'page-shell.html' %}
```
Comment (jbothma) [inline _layouts/about.html]: Please move the layouts files to the templates folder like you'd normally have in a django app.

File: _layouts/contributed-data.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (jbothma) [inline _layouts/contributed-data.html]: please delete the jekyll templates that are not used yet. We will copy them over as we get to each page. Unused code will just get stale and confusing.

File: assets/generated/styles.bundle.css [hunk 6]
```diff
@@ -0,0 +1,5030 @@
+html {
```
Comment (jbothma) [inline assets/generated/styles.bundle.css]: Generated budnles should not be checked into git. They should be build as needed in development. In production they will be built by the node-js heroku buildpack and then the python buidpack will run django collectstatic.

File: _data/2018-19/manual_resources.yaml (hunk 2 above)
Comment (havanhuy1997) [reply _data/2018-19/manual_resources.yaml]: _data includes the videos.yaml about.yaml navbar.yaml, and the about page needs these data now. I used the PyYAML to parse these data to django context

**Comment by jbothma** [comment]
ok - but please delete the stuff you don't need - it's hundreds of files
that we don't need in this project. Only keep the files we need.

**Pull request closed without merging**